src/test/java/StationConnectionTest.java

```diff
@@ -11,11 +17,17 @@
     public void getConnections() throws IOException
     {
         //given
+        Gson gson = new Gson();
+        Reader reader = Files.newBufferedReader(Paths.get("src/main/resources/SubwayLines.json"));
+        Reader readerStation = Files.newBufferedReader(Paths.get("src/main/resources/Subwaystations.json"));
         Integer lines[] = {359, 94};
```

Comment:
There is no reason to have this as an array. Just put the values in the assertEquals